### PR TITLE
Fix sum multiply

### PR DESCRIPTION
**Pull request by narekhovhannisyan**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- Fixes implementation logic of sum/multiply
-

### CODE REVIEW

File: src/lib/multiply/index.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ export const Multiply = (globalConfig: MultiplyConfig): PluginInterface => {
     kind: 'execute',
   };
 
-  /**Checks global confiog value are valid */
+  /**
+   * Checks global confiog value are valid.
```
Comment (manushak) [inline src/lib/multiply/index.ts:17]: typo in `confiog`

File: src/lib/multiply/index.ts [hunk 2]
```diff
@@ -13,7 +13,9 @@ export const Multiply = (globalConfig: MultiplyConfig): PluginInterface => {
...
+   */
   const validateGlobalConfig = () => {
     if (inputParameters.length === 0) {
```
Comment (manushak) [inline src/lib/multiply/index.ts:20]: you can use `zod` as you suggested, like
const configSchema = z.object({
  inputParameters: z.array(z.string()).gt(1),
  outputParameter: z.string().nonempty(),
});

File: src/lib/sum/index.ts [hunk 3]
```diff
@@ -60,28 +67,25 @@ export const Sum = (globalConfig: SumConfig): PluginInterface => {
...
     validateGlobalConfig();
-    inputs.map(input => {
+
+    return inputs.map(input => {
       const safeInput = validateSingleInput(input);
-      return calculateSum(safeInput, inputParameters, outputParameter);
+
+      return {
+        ...safeInput,
+        [outputParameter]: calculateSum(safeInput, inputParameters),
```
Comment (manushak) [inline src/lib/sum/index.ts:76]: `inputParameters` is a global variable

File: src/lib/multiply/index.ts (hunk 1 above)
Comment (narekhovhannisyan) [reply src/lib/multiply/index.ts:17]: it's @jmcook1186 comment :D

Comment (jmcook1186) [reply src/lib/multiply/index.ts:17]:
```suggestion
   * Checks global config value are valid.
```

File: src/lib/sum/index.ts [hunk 4]
```diff
@@ -14,6 +14,9 @@ export const Sum = (globalConfig: SumConfig): PluginInterface => {
     kind: 'execute',
   };
 
+  /**
+   * Checks global confiog value are valid.
```
Comment (jmcook1186) [inline src/lib/sum/index.ts:18]:
```suggestion
   * Checks global config value are valid.
```

**Review by jmcook1186 — APPROVED** [review]
thanks!

**Pull request merged**